pset-functions.js

```diff
@@ -208,6 +346,15 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+var texts = ["loop", "rune", "apple"];
+
+
+for(var idx in texts){
+    var text = texts[idx].split("");
+    console.log(text.join("") + " is " + text.some(function(v,i,a){return a.lastIndexOf(v)!=i;}));
+
+  }
```

Comment:
@jl1982 Good work here on this solution. I see your output is correct and you're getting the right outcome.
Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L291)